### ISSUE TITLE
syslog-ng shows warning constantly

### ISSUE THREAD

**Issue by amosbird** [phusion/baseimage-docker#476]
Hi, is this harmful?

```
syslog-ng[11]: WARNING: you are using the pipe driver, underlying file is not a FIFO, it should be used by file(); filename='/dev/stdout'
```

**Comment by hyperknot** [comment]
No, it's just a warning and will be fixed in 0.10.1.

**Comment by amosbird** [comment]
great. thanks!

**Comment by R0GGER** [comment]
Same issue here...
`Mar 20 15:52:22 86fcf2e687d6 syslog-ng[11]: WARNING: you are using the pipe driver, underlying file is not a FIFO, it should be used by file(); filename='/dev/stdout'`

When will v0.10.1 be released?

**Comment by hyperknot** [comment]
@Theaxiom do you mind if I release 0.10.1?

**Comment by Theaxiom** [comment]
I'll release it tonight.

**Comment by R0GGER** [comment]
Any update yet?

**Comment by Theaxiom** [comment]
I have released version `0.10.1` I apologize for the delay! 👍
